lib/utils.js

```diff
@@ -120,5 +127,6 @@ class Utils {
 exports.Utils = Utils;
 Utils.USER_AGENT = 'setup-jfrog-cli-github-action/' + require('../package.json').version;
 Utils.SERVER_TOKEN_PREFIX = /^JF_ARTIFACTORY_.*$/;
+Utils.NEW_CONFIG_CLI_VERSION = '1.45.0';
```

Comment:
I suggest we add a comment, explaining why this version is significant, and what this constant is used for.